.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
you are `.gitignore`-ing this, but even so, you still need to remove it from your git history.

```shell
git rm .vscode/settings.json
```